Dockerfile

```diff
@@ -1,19 +1,17 @@
-FROM quay.io/ukhomeofficedigital/openjdk11:v11.0.5_10
+FROM quay.io/ukhomeofficedigital/alpine:3.10
```

Comment:
I wonder wether we should have a new base image that has the health check in it?

Reply:
My personal preference would be to have the clean base and build on it like this but happy to be overruled, I think images with whatever added are hard to interpret for future people

Reply:
lets leave it clean. Shame we have to add the healthcheck to 9 repos!!!